module1/homework/grayscaleImages/main.cpp

```diff
@@ -1,7 +1,7 @@
 #include <array>
 #include <forward_list>
 
-// TODO: include
+#include "compression.hpp"
```

Comment:
You can remove array here since it's in the header.

Reply:
All of our header files have #pramga once, so it protect us from multiply includes.

Reply:
But still there is no reason to keep this include here.